contracts/src/token/erc1155/mod.rs

```diff
@@ -786,7 +780,7 @@ impl Erc1155 {
     /// interface id or returned with error, then the error
     /// [`Error::InvalidReceiver`] is returned.
     fn _check_on_erc1155_received(
-        &mut self,
+        &self,
```

Comment:
Don't forget to update the documentation above each function for which the signature has changed

Reply:
Thanks for the reminder, @0xNeshi. I've updated the docs for each function where the signature has changed!